internal/server/v1/observations/golden/bulk_point_linked_test.go

```diff
@@ -41,31 +41,31 @@ func TestBulkObservationsPointLinked(t *testing.T) {
 			date         string
 			goldenFile   string
 		}{
-			{
-				"County",
-				"geoId/06",
-				[]string{"Count_Person", "Median_Age_Person"},
-				"",
-				"CA_County.json",
-			},
-			{
-				"County",
-				"country/USA",
-				[]string{"Count_Person", "Median_Age_Person", "NumberOfMonths_WetBulbTemperature_35COrMore_RCP45_MinRelativeHumidity"},
-				"",
-				"USA_County.json",
-			},
+			// {
```

Comment:
Want to delete instead of comment?

Reply:
I need to talk to Francesco about this flakiness. We do want to uncomment this when he has a proper fix.